addons/web/static/tests/views/fields/properties_field_tests.js

```diff
@@ -565,12 +565,35 @@ QUnit.module("Fields", (hooks) => {
         await click(target, ".o_field_property_selection_option:nth-child(2) .fa-trash-o");
         options = popover.querySelectorAll(".o_field_property_selection_option");
         assert.strictEqual(options.length, 3, "Should have removed the second option");
-        const optionValues = [...options].map((option) => option.querySelector("input").value);
+        let optionValues = [...options].map((option) => option.querySelector("input").value);
         assert.deepEqual(
             optionValues,
             ["A", "C", "New option"],
             "Should have removed the second option"
         );
+
+        // focus should be in option C
+        assert.strictEqual(document.activeElement, options[1].querySelector("input"));
+        // press enter to add an option after the second one
+        await triggerEvent(
+            target,
+            ".o_field_property_selection_option:nth-child(2) input",
+            "keydown",
+            { key: "Enter" }
+        );
+        await editInput(
+            target,
+            ".o_field_property_selection_option:nth-child(3) input",
+            "New option 2"
+        );
+        options = popover.querySelectorAll(".o_field_property_selection_option");
+        assert.strictEqual(options.length, 4, "Should have added a new option");
+        optionValues = [...options].map((option) => option.querySelector("input").value);
```

Comment:
Not sure we need a variable after all, you could just inline this inside deepEqual (same for above).